api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/TXTBCohortQueries.java

```diff
@@ -828,4 +840,126 @@ public CohortDefinition txTbDenominator() {
     addGeneralParameters(cd);
     return cd;
   }
+
+  public CohortDefinition positiveScreening() {
+    CompositionCohortDefinition cd = new CompositionCohortDefinition();
+    CohortDefinition A = codedYesTbScreening();
+    addGeneralParameters(A);
+    cd.addSearch("A", map(A, codedObsParameterMapping));
+    CohortDefinition B = positiveOrNegativeInvesitionResult();
+    addGeneralParameters(B);
+    cd.addSearch("B", map(B, generalParameterMapping));
+    CohortDefinition C = tbTreatmentStartDateWithinReportingDate();
+    addGeneralParameters(C);
+    cd.addSearch("C", map(C, generalParameterMapping));
```

Comment:
How is the map different from the usual method from the EptsReportingUtils.map()?

Reply:
@kaweesi here is one